cupy/core/raw.pyx

```diff
@@ -366,7 +376,8 @@ cdef class RawModule:
         ker = RawKernel(
             self.code, name, self.options, self.backend,
             translate_cucomplex=self.translate_cucomplex,
-            enable_cooperative_groups=self.enable_cooperative_groups)
+            enable_cooperative_groups=self.enable_cooperative_groups
```

Comment:
Caught an obvious error 😂, not sure if this is the only one...
```suggestion
            enable_cooperative_groups=self.enable_cooperative_groups,
```

Reply:
HA! Thanks, I had just found it :joy:

Reply:
Thanks for taking to time to help me! Going to do some more testing before my next push.

Reply:
Whenever I touch `raw.pyx` and/or `compiler.py` I'd always run `purest tests/cupy_tests/core_tests/test_raw.py` as a quick check to ensure I don't mess up 😁

Right, I wonder about NVRTC. Perhaps the warning log is generated in the log, fetched by `nvrtcGetProgramLog`? Then we will need to examine the log when `log_stream is not None`?

Reply:
All tests failed. :facepalm: 
I'll take a look at it in the morning.